src/core_modules/capture-core/components/FormFields/Options/SelectVirtualizedV2/OptionsSelectVirtualized.component.js

```diff
@@ -53,6 +53,7 @@ type Props = {
         selectedIconLeftContainer: string,
         selectedIconRightContainer: string,
     },
+    dataTest?: string,
```

Comment:
I couldnt find where this is being passes as a property when we use the `OptionsSelectVirtualized.component`. 

Either I couldnt find it or we might not be using it. Can you help me find out?

Reply:
Sure, it's used [here](https://github.com/dhis2/capture-app/blob/0abe0991ced68e53d451fd7b9142a597b1a9a34d/src/core_modules/capture-core/components/Pages/NewRelationship/RegisterTei/RegistrationSection/ProgramSelector/ProgramSelector.component.js#L35)